### ISSUE TITLE
Glossary terms do not (and cannot?) include a word wrap or break

### ISSUE THREAD

**Issue by tw77** [pressbooks/pressbooks#3479]
For glossary terms, there is no word wrap or break. So if you have a long phrase as a glossary term, it will force a lot of white space if the string within the glossary shortcode is too long to fit on its current line:

![image](https://github.com/pressbooks/pressbooks/assets/44182629/ac021031-c01e-4603-bea9-e456678e3e8e)

The text within the glossary shortcode can be styled through the .glossary-term class (as demonstrated in image using text-transform to uppercase the string). Even though this is the correct class, though, it doesn't respond to word-wrap, overflow-wrap, or word-break.

**Comment by SteelWagstaff** [comment]
I don't think this is solvable with the `<button>` element. We might have to make the glossary term element an `<a>` tag or similar HTML element to allow it to be displayed in-line with surrounding text that spans multiple lines.
